### DISCUSSION TITLE
UVR5 and AMD Cards

### DISCUSSION THREAD

**Discussion by celeste-ex3** [Anjok07/ultimatevocalremovergui#1032] [General]
So the thing here is that there is another version of UVR5 other than the Main Download Called The OpenCl version which claims to be compatible with AMD Cards And Intel I guess, but the real question here is that I am an owner of an AMD Radeon Rx 470 4Gb card and I tried the OpenCl version and when I selected the GPU Conversion Option My Pc would Blackout and then crash, aftrewards, later when I will turn my pc on again and try to run the program a pop up will appear showing some error I don't have the screenshot as I later switched to the main version to check if my pc is even compatible with it or not, but works fine with CPU Conversion, So my appeal here is to the Creator of this astonishing software please do something about this or if someone have any suggestions on how to fix it please feel free to let me know. (NOTE: I know my GPU aint that good but still it's average and better than my CPU)

**Comment by saulop** [comment]
Maybe the memory is too small. Try lower values for the "segment size" and "overlap". Good luck. I use OpenCl version and it is great!

**Comment by jt66335** [comment]
i am also having the same problem and i am using rx580 2304sp 8gb version

**Comment by TitanGames-729** [comment]
It was really surprising for me, as i should have had at least 99.9 errors, however, it worked. I have a rx 580 8gb sapphire nitro+. I installed the main first then the patch. Took me around 45 seconds to process

Upd: The input was 3:43 long

Upd: Aaaand it stopped working, throwing always an error. What i did is install other python version since i needed it but i had like 2 of them. Maybe that is the problem. Also, i tried to reinstall it but not using the patch.

**Comment by TitanGames-729** [comment]
I managed to fix it for me. I checked the settings and found that my gpu was being detected as RX580 [ZLUDA]. Fortunately, i was in process of it's uninstallation as ZLUDA was useless for me. I uninstalled ROCm and deleted all folders, deleted ZLUDA from C:/ and removed all related paths in PATH. After one reboot i started UVR5 and now the GPU is detected as RX 580 [0] and UVR5 works as needed. I guess there is no support for ZLUDA at the moment and maybe it will be added in future. Also, I think it would have been great to open an another issue to inform the devs.

**Reply by Jaddrrz** [reply]
Hey, I was looking for this version that works on AMD gpus but the link on their repo doesn't work anymore. Is there any way I could get my hands on it?

**Reply by EddyFler** [reply]
Anyone did find the link for download amd version UVR?? I downloaded beta version but amd GPU conversion outputs me error...

**Reply by deton24** [reply]
You don't need that DirectML fork, as it was integrated with newer UVR versions, indeed. The error you have would probably occur in that fork anyway. 
If it's an error indicating insufficient GPU memory on Roformers, decrease chunk_size in the model yaml/config to e.g. 112455 (on old patches to dim_t = 256) - it's the lowest working on 4GB VRAM Intel/AMD GPUs and 900MB Roformers.
